2023/AzureBengaluru/data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
```

Comment:
We need the GitHub ID's of those users who are allowed to edit your folder! For example yours @Keshav1308